system/Debug/Toolbar/Collectors/Events.php

```diff
@@ -63,7 +63,7 @@ class Events extends BaseCollector
      */
     public function __construct()
     {
-        $this->viewer = Services::renderer();
+        $this->viewer = Services::renderer(null, null, false);
```

Comment:
Would it be clearer to use `single_service('renderer')` here?

Reply:
Thanks. Using not shared service was wrong. Because the shared View has the performance data.
I will fix.

Reply:
Done.